trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -9,10 +10,38 @@ const createFacilityAmendment = async (req, res) => {
   return res.status(422).send({ data: 'Unable to create amendment' });
 };
 
+const sendAmendmentEmail = async (amendmentId, facilityId) => {
```

Comment:
Can we have the try block extended to the top, this in case there is a connection error or deal not found in the DB it will not cause TMF API to crash and will rather throw an error in the console.

Reply:
done